### PR TITLE
kore-repl: In the graph visualizer add no. of nodes on dotted edges

### LINKED ISSUE #2902: kore-repl: in the graph visualiser add no. of nodes on dotted edges

### PR DESCRIPTION
**Pull request by andreiburdusa**
Fixes #2902

### Estimate:
- December 16

---

###### Review checklist

The author performs the actions on the checklist. The reviewer evaluates the work and checks the boxes as they are completed.

-   [ ] **Summary.** Write a summary of the changes. Explain what you did to fix the issue, and why you did it. Present the changes in a logical order. Instead of writing a summary in the pull request, you may push a clean Git history.
-   [ ] **Documentation.** Write documentation for new functions. Update documentation for functions that changed, or complete documentation where it is missing.
-   [ ] **Tests.** Write unit tests for every change. Write the unit tests that were missing before the changes. Include any examples from the reported issue as integration tests.
-   [ ] **Clean up.** The changes are already clean. Clean up anything near the changes that you noticed while working. This does not mean only spatially near the changes, but logically near: any code that interacts with the changes!

### CODE REVIEW

**Comment by andreiburdusa** [comment]
The graph for `tiny/a-to-c-spec.k` would look like
Graph

File: kore/src/Kore/Repl/Interpreter.hs [hunk 1]
```diff
@@ -1521,9 +1521,13 @@ graphParams gr =
             ]
         }
   where
+    nrOfNodes :: Natural -> Text.Lazy.Text
```
Comment (ana-pantilie) [inline kore/src/Kore/Repl/Interpreter.hs:1524]: Why does this use lazy text?

Comment (andreiburdusa) [reply kore/src/Kore/Repl/Interpreter.hs:1524]: This is what [`Graph.textLabel`](https://hackage.haskell.org/package/graphviz-2999.20.1.0/docs/Data-GraphViz-Attributes.html#v:textLabel) uses.

**Review by dopamane — APPROVED** [review]
:+1:

**Pull request merged**